components/webpack/dev.config.js

```diff
@@ -30,7 +26,7 @@ module.exports = {
     rules: [
       {
         test: /\.tsx?$/,
-        loader: 'awesome-typescript-loader'
+        loader: 'ts-loader'
```

Comment:
I had quite some problems with awesome loader, because of brave-browser -> brave-core relation and files were not found. With ts-loader everything is working ok